packages/core/src/main/Account.ts

```diff
@@ -349,14 +335,15 @@ export class Account<T = any> extends EventEmitter {
     });
 
     const clientService = new ClientService(this.apiClient, this.storeEngine, cryptographyService);
+    const mlsService = new MLSService(this.mlsConfig, this.apiClient, () => this.coreCryptoClient);
```

Comment:
Maybe (in a follow up PR, not this one) we could move **all** code that uses coreCrypto to the MLS scope :)

Reply:
Yeah, this is what we aim for! 😄